nibiru/query_clients/util.py

```diff
@@ -1,10 +1,24 @@
+from typing import Dict, List
+
+from google.protobuf import message as protobuf_message
 from google.protobuf.json_format import MessageToDict
 
-from nibiru.utils import from_sdk_dec
+from nibiru.utils import from_sdk_dec, from_sdk_int
+
+PROTOBUF_MSG_BASE_ATTRS: List[str] = (
+    dir(protobuf_message.Message)
+    + ['Extensions', 'FindInitializationErrors', '_CheckCalledFromGeneratedFile']
```

Comment:
Are we sure this does not change from one proto to another?

Reply:
As far as I've seen, the protobuf messages all have these attributes because they are built from the `protobuf.message.Message` base class.